packages/mobile/src/screens/splash-screen/SplashScreen.tsx

```diff
@@ -1,120 +1,96 @@
-import { useRef, useState, useEffect, useCallback } from 'react'
+import { useState, useRef, useEffect } from 'react'
 
-import { accountSelectors, themeSelectors, Status } from '@audius/common'
-import LottieView from 'lottie-react-native'
-import { StyleSheet, Animated, StatusBar, Platform } from 'react-native'
-import { useSelector } from 'react-redux'
-import { useEffectOnce } from 'react-use'
+import { Animated, StyleSheet } from 'react-native'
+import * as BootSplash from 'react-native-bootsplash'
 
-import { updateStatusBarTheme } from 'app/utils/theme'
+import SplashLogo from 'app/assets/images/bootsplash_logo.svg'
+import { makeStyles } from 'app/styles'
+import { zIndex } from 'app/utils/zIndex'
 
-const { getAccountStatus } = accountSelectors
-const { getTheme, getSystemAppearance } = themeSelectors
+/**
+ * Assets for this splash screen are generated with
+ * npx react-native generate-bootsplash \
+ *  src/assets/images/bootsplash_logo.png \
+ *  --background-color=7E1BCC \
+ *  --logo-width=150 \
+ *  --assets-path=src/assets/images
+ */
 
-const SCALE_TO = 1.2
-const ANIM_DURATION_MS = 2000
-const LOTTIE_HEIGHT = 1350
-const BACKGROUND_COLOR = '#7E1BCC'
+// Extra larger render width so when we scale it up,
+// resolution is maintained.
+const RENDER_WIDTH = 1000
+const START_SIZE = 0.15
+const END_SIZE = 1
 
-const styles = StyleSheet.create({
-  container: {
-    backgroundColor: BACKGROUND_COLOR,
-    position: 'absolute',
-    top: 0,
-    left: 0,
-    right: 0,
-    bottom: 0,
-    zIndex: 10,
-    alignItems: 'center',
-    justifyContent: 'center'
+const useStyles = makeStyles(({ palette }) => {
+  return {
+    splash: {
+      zIndex: zIndex.SPLASH_SCREEN,
+      flex: 1,
+      justifyContent: 'center',
+      alignItems: 'center',
+      backgroundColor: palette.secondary
+    },
+    logo: {
+      width: RENDER_WIDTH
+    }
   }
 })
 
-export const SplashScreen = () => {
-  const [animationFinished, setAnimationFinished] = useState(false)
-  const accountStatus = useSelector(getAccountStatus)
-  const theme = useSelector(getTheme)
-  const systemAppearance = useSelector(getSystemAppearance)
-  const backgroundOpacityAnim = useRef(new Animated.Value(1))
-
-  useEffectOnce(() => {
-    if (Platform.OS === 'ios') {
-      // Hide the StatusBar on ios
-      updateStatusBarTheme(theme, systemAppearance)
```

Comment:
We don't have to do anything with the status bar colors anymore? Even on android 👀

If so that's pretty awesome

Can you test starting the app with different default OS themes enabled? I think the splashscreen was responsible for the initial setting of the status bar color

Reply:
It seems to work fine! Unless I'm missing something ⁉️

Reply:
Cool, it's probably fine!

Reply:
You were right. Updated.